### DISCUSSION TITLE
Icinga Monitoring

### DISCUSSION THREAD

**Discussion by ChrLau** [selfhst/icons#97] [New Icon Request]
Hi,

currently there is no logo for the Icinga Monitoring software.

Press & Media guidelines regarding logo usage and logo kit: https://icinga.com/company/press-media/

Logo Kit: https://icinga.com/wp-content/uploads/2021/06/Icinga-Logo-Bundle.zip
Icinga Logo Guidelines: https://icinga.com/wp-content/uploads/2021/06/Icinga-Logo-StyleGuide.pdf

**Comment by shollyethan** [comment]
Icinga has been added - thanks!
